azurerm/internal/services/iothub/registration.go

```diff
@@ -21,6 +21,7 @@ func (r Registration) WebsiteCategories() []string {
 // SupportedDataSources returns the supported Data Sources supported by this Service
 func (r Registration) SupportedDataSources() map[string]*schema.Resource {
 	return map[string]*schema.Resource{
+		"azurerm_iothub_digital_twins":            dataSourceDigitalTwins(),
```

Comment:
THis is a digital twin what? or will there not be any other resources for digital twins? `azurerm_digital_twins_??????` instance?

Reply:
Fixed in PR #9430